src/PDM.Core/PDM.Core.csproj

```diff
@@ -1,20 +1,39 @@
-<Project Sdk="Microsoft.NET.Sdk">
-
-	<PropertyGroup>
-		<TargetFramework>net6.0</TargetFramework>
-		<ImplicitUsings>enable</ImplicitUsings>
-		<Nullable>enable</Nullable>
-		<LangVersion>latest</LangVersion>
-		<AnalysisLevel>latest-all</AnalysisLevel>
-		<RootNamespace>PDM</RootNamespace>
-	</PropertyGroup>
-
-	<ItemGroup>
-		<PackageReference Include="Google.Protobuf" Version="3.22.0" />
-		<PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="7.0.0" />
-		<PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="6.0.0" />
-		<PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="6.0.0" />
-		<PackageReference Include="System.Linq.Dynamic.Core" Version="1.2.25" />
-	</ItemGroup>
-
-</Project>
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<Authors>Barry S. Stahl</Authors>
+		<Company>Protobuf Data Mapper</Company>
+		<Copyright>Copyright 2023</Copyright>
+		<Description>A library for mapping and transforming one Protobuf message into another</Description>
+		<RepositoryUrl>https://github.com/bsstahl/PDM</RepositoryUrl>
+		<ProjectGuid>{76C2AB7D-12E6-40D6-A87E-71FF2A8F23D7}</ProjectGuid>
+		<PackageId>ProtobufDataMapper</PackageId>

```

Comment:
Do we maybe want to change the root namespace in the package to align with this name, instead of PDM?

Reply:
I don't feel like it is necessary but am willing if it is the right thing to do. I only changed the package name because the name PDM was already used in NuGet (not surprising really). I still feel like it is good as the root namespace within the package though.